src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/GlyphTypeface.cs

```diff
@@ -251,7 +243,6 @@ public Geometry GetGlyphOutline(ushort glyphIndex, double renderingEmSize, doubl
         [CLSCompliant(false)]
         public byte[] ComputeSubset(ICollection<ushort> glyphs)
         {
-            SecurityHelper.DemandUnmanagedCode();
             DemandPermissionsForFontInformation();
```

Comment:
do we need `DemandPermissionsForFontInformation` anymore?

Reply:
No we don't, but i've already removed this in a separate branch (which will be the last PR I open as the purpose of this PR was more targeted to internal permission types and `SecurityHelper`)